docs/autofix.md

```diff
@@ -0,0 +1,30 @@
+# Autofix (autofix)
```

Comment:
I think this is one of the most awesome features we have! I think we want to hype this a little more 😄

Reply:
I agree! Do you have any suggestions?

Reply:
How about showing how it keeps comments and how it fixes a basically empty file into a secure one